src/v2/Apps/Artwork/Components/ArtworkMeta.tsx

```diff
@@ -111,6 +125,40 @@ export class ArtworkMeta extends Component<ArtworkMetaProps> {
     )
   }
 
+  renderZendeskScript() {
+    const {
+      is_acquireable,
+      is_in_auction,
+      isInquireable,
+      isOfferable,
+      listPrice,
+      priceCurrency,
+    } = this.props.artwork
+
+    const BNMO_CURRENCY_THRESHOLDS = {
+      USD: 10000,
+      EUR: 8000,
+      HKD: 77000,
+      GBP: 7000,
+    }
+
+    // This accounts for exact price and price ranges
+    const artworkPrice = listPrice?.major
+      ? listPrice?.major
+      : listPrice?.minPrice?.major
```

Comment:
Are these `?` necessary?

Reply:
Maybe on the one after `minPrice`. I'll remove the others.